saskatoon/member/models.py

```diff
@@ -109,8 +112,9 @@ def is_onboarding(self):
         # Retrieve only name fields from QuerySet of Groups
         group_names = [g.name  for g in self.role_groups]
         return ('pickleader' not in group_names and
-            'volunteer' in group_names and
-            self.password != '')
+                'volunteer' in group_names and
+                self.has_temporary_password and
+                self.password != '')
```

Comment:
if `self.has_temporary_password` is `True`, wouldn't that imply that `self.password != ''` ? 

```
return (
    'pickleader' not in group_names and
    'volunteer' in group_names and
    self.has_temporary_password
)
```

Reply:
true, i just kept stacking conditions.